bootcfg/http/ignition.go

```diff
@@ -88,15 +88,23 @@ func (s *Server) ignitionHandler(core server.Server) ContextHandler {
 			return
 		}
 
-		// Parse fuze config into an Ignition config
-		config, err := fuze.ParseAsV2_0_0(buf.Bytes())
-		if err == nil {
-			s.renderJSON(w, config)
+		// Parse bytes into a Fuze Config
+		config, report := fuze.Parse(buf.Bytes())
+		if report.IsFatal() {
+			s.logger.Errorf("error parsing Fuze config: %s", report.String())
```

Comment:
It might be better to print the report to its own line since the report itself can be multiple lines.